### PR TITLE
Set TargetsMobile=true on Browser TargetOS and build the wasm runtimes earlier during the libs build

### PR DESCRIPTION
**Pull request by akoeplinger**
Ensures the wasm runtimes end up in lib-runtime-packs and the runtime pack nuget.

### CODE REVIEW

**Review by vargaz — APPROVED** [review]

**Review by steveisok — APPROVED** [review]

File: src/libraries/src.proj [hunk 1]
```diff
@@ -46,4 +46,12 @@
              Properties="$(TraversalGlobalProperties)" />
   </Target>
 
+  <!-- Wasm runtimes depend on the mono runtime and libs to be built, so they can only be built here -->
+  <Target Name="BuildWasmPackageDependency"
+          Condition="'$(TargetOS)' == 'Browser'"
+          AfterTargets="Build">
+    <MSBuild Projects="$(MonoProjectRoot)\wasm\wasm.proj"
```
Comment (ViktorHofer) [inline src/libraries/src.proj:53]: Why are we building a mono project from within libraries?

Comment (akoeplinger) [reply src/libraries/src.proj:53]: WASM is a bit special, it needs to run a processor over the whole BCL to find p/invokes and generate a table with names and signatures from them. This seemed like a good place to do that.

Comment (ViktorHofer) [reply src/libraries/src.proj:53]: Wouldn't it make more sense to do that in a mono subset instead which runs after libraries?

Comment (akoeplinger) [reply src/libraries/src.proj:53]: Sure, if you show me how to do that :)

Comment (akoeplinger) [reply src/libraries/src.proj:53]: Sure, if you show me how to do that :)

Comment (ViktorHofer) [reply src/libraries/src.proj:53]: I'm fine if you do that in a follow-up if you prefer. I will submit a PR make it easier to run after some subsets.

**Pull request merged**